aws-lc-rs/src/cipher/padded.rs

```diff
@@ -390,7 +429,13 @@ mod tests {
 
                 let context = encrypting_key.less_safe_encrypt(&mut in_out, ec).unwrap();
 
-                assert_eq!(expected_ciphertext, in_out);
+                if ($padding == PaddingStrategy::ISO10126) {
+                    // This padding scheme is technically non-deterministic in nature if the padding is more then one
```

Comment:
For unknown input to the padding decoder, this assumption is true.
But for your implementation you can make a stronger assumption and testing doesn't take unknown inputs.
Hence, does one want a test to preserve current implementation behaviour or let it be general.

I have 0 strong opinion on it.